crates/oxc_linter/src/autofix/fixer.rs

```diff
@@ -39,25 +20,26 @@ impl<'a> Fixer<'a> {
             let source_text = self.source_text;
             let mut output = String::with_capacity(source_text.len());
             // To record the position of the last fix.
-            let mut last_pos = 0;
+            let mut last_pos: i64 = -1;
```

Comment:
We have `None` in Rust ;-)

Reply:
We can even go for `Option<std::num::NonZeroU32>` if you are a purist.

Reply:
I tried `Option` with `unwrap_or(0)`, but it would cause some tests fail because of the `start <= last_pos.unwrap_or(0)`. However, we cannot change it to `start < last_post.unwrap_or(0)` since it would cause `apply_one_fix_when_the_start_the_same_as_the_previous_end` test fail.

> Option<std::num::NonZeroU32> 

We cannot use this here because the last_pos may be a zero.